### PR TITLE
Validate the email address is a gov.uk one

### PR DESCRIPTION
**Pull request by andreagrandi**

### CODE REVIEW

File: sandboxmgt/app/forms.py [hunk 1]
```diff
@@ -1,6 +1,17 @@
 from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
 from .models import Request
 
+
+def validate_gov_email(value):
+    if value.endswith('gov.uk') is False:
```
Comment (davidread) [inline sandboxmgt/app/forms.py]: or even better ".gov.uk"?

File: sandboxmgt/app/forms.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+        raise ValidationError(
+            _('%(value)s is not a valid gov.uk email address'),
```
Comment (davidread) [inline sandboxmgt/app/forms.py]: interesting, what's the translatability for?

**Review by davidread — COMMENTED** [review]
Looks great. Just a couple of q's.

File: sandboxmgt/app/forms.py (hunk 1 above)
Comment (andreagrandi) [reply sandboxmgt/app/forms.py]: right!

File: sandboxmgt/app/forms.py (hunk 2 above)
Comment (andreagrandi) [reply sandboxmgt/app/forms.py]: ehm... no idea, I used/adapted an example from Django documentation

**Pull request merged**